rules-catalogue/rules-catalogue.md

```diff
@@ -17,9 +17,9 @@
 | azure-java-version-02000 | Java version found to be lower than JAVA_8 | mandatory | 3 | X | X | X | X | azure-java-version.windup.xml |
 | azure-keystore-certificates-01000 | Java KeyStore file found | potential | 3 | X | X | X | X | azure-keystore-certificates.windup.xml |
 | azure-keystore-certificates-02000 | The application loads certificates into a KeyStore | potential | 3 | X | X | X | X | azure-keystore-certificates.windup.xml |
-| azure-mq-config-kafka-01000 | Kafka connection string, username or password found in configuration file | information | 0 | X | X | X | X | azure-mq-config.windup.xml |
```

Comment:
@KaiqianYang this file is autogenerated. Can you please revert these changes?

Reply:
> @KaiqianYang this file is autogenerated. Can you please revert these changes?

@brunoborges Reverted: https://github.com/Azure/appcat-rulesets/commit/1611110794315ac2b24dd30cfbe1d1cd7947366a